content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -64,29 +64,40 @@ spec:
           gpu-vendor.example/example-gpu: 1 # requesting 1 GPU
 ```
 
-## Clusters containing different types of GPUs
+## Automatically labeling nodes with Node Feature Discovery {#node-feature-discovery}
 
-If different nodes in your cluster have different types of GPUs, then you
-can use [Node Labels and Node Selectors](/docs/tasks/configure-pod-container/assign-pods-nodes/)
-to schedule pods to appropriate nodes.
+As and administrator, you can automatically discover and label all your GPU enabled nodes
+by deploying the K8S-Sig project Node Feature Discovery [NFD](https://github.com/kubernetes-sigs/node-feature-discovery).
+NFD enables node feature discovery for Kubernetes. 
+It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels and optionally node extended resources, annotations and node taints. 
```

Comment:
L#71 doesn't have as much info payload as #72, and I think we can combine:

"NFD detects the hardware features that are available on each node in a Kubernetes cluster and advertises those features. Typically, NFD adds node labels to advertise the features, but NFD can also add extended resources, annotations, and node taints."